modules/ROOT/partials/DEPRECATED/spellchecker.adoc

```diff
@@ -1 +1 @@
-IMPORTANT: The *free* {productname} Spell Checker plugin (`spellchecker`) was deprecated with the release of {productname} 5.4. For details, see link:{baseurl}/release-notes/release-notes54/#thefreetinymcespellcheckerplugin[the free {productname} Spell Checker plugin deprecation notice]. The free Spell Checker plugin will be removed in {productname} 6.0.
+IMPORTANT: The *free* {productname} Spell Checker plugin (`spellchecker`) was deprecated with the release of {productname} 5.4. For details, see xref:release-notes/release-notes54.adoc#thefreetinymcespellcheckerplugin[the free {productname} Spell Checker plugin deprecation notice]. The free Spell Checker plugin will be removed in {productname} 6.0.
```

Comment:
This xref doesn't resolve as the `thefreetinymcespellcheckerplugin` id doesn't exist. It looks like an id needs to be added to the heading in the 5.4 release notes.

- [ ] Fix broken xref

Reply:
Done! I added the missing id